Classes/Emogrifier.php

```diff
@@ -289,7 +289,7 @@ public function emogrify()
 
         $this->copyCssWithMediaToStyleNode($cssParts, $xmlDocument);
 
-        return mb_convert_encoding($xmlDocument->saveHTML(), self::ENCODING, 'HTML-ENTITIES');
+        return $xmlDocument->saveHTML();
```

Comment:
I think the constant ENCODING then is not used anymore either, and we can remove it as well.